src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -1646,14 +1644,12 @@ public void addTab(BasePanel bp, File file, boolean raisePanel) {
                 title = title + '*';
             }
         } else {
-            title = file.getName();
+            title = file.getName() + " (" + bp.database.getEntryCount() + ")";
```

Comment:
Of course this should be done. I just don't know if the majority wants this feature so just demoing it here.